jwql/website/jwql_proj/settings.py

```diff
@@ -95,20 +95,21 @@
 ]
 
 MESSAGE_TAGS = {
-        messages.DEBUG: 'alert-secondary',
-        messages.INFO: 'alert-info',
-        messages.SUCCESS: 'alert-success',
-        messages.WARNING: 'alert-warning',
-        messages.ERROR: 'alert-danger',
- }
+    messages.DEBUG: 'alert-secondary',
+    messages.INFO: 'alert-info',
+    messages.SUCCESS: 'alert-success',
+    messages.WARNING: 'alert-warning',
+    messages.ERROR: 'alert-danger',
+}
 
 WSGI_APPLICATION = 'jwql.website.jwql_proj.wsgi.application'
 
 
 # Database
 # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
 DATABASES = {
-    'default': get_config()['django_database']
+    'default': get_config()['django_databases']['default'],
```

Comment:
in my local config the key is `django_database` without the `s`. Did we update the key at some point? Also are `monitors` and `default` new keys we need to add?

Reply:
So, I changed it from `django_database` to `django_databases` because there are now 2 django databases – the existing one (both previously and now called "default"), and the JWQLDB one, called "monitors".